### PR TITLE
Adding Batch TaskReactivate to swagger spec

### PR DESCRIPTION
**Pull request by matthchr**

### CODE REVIEW

File: batch/2016-07-01.3.1/swagger/BatchService.json [hunk 1]
```diff
@@ -8253,6 +8253,156 @@
...
+                "description": "The content of the Last-Modified HTTP response header.",
+                "type": "string",
+                "format": "date-time-rfc1123"
+              },
+              "DataServiceId": {
+                "description": "The OData id of the resource to which the request applied.",
+                "type": "string"
+              }
+            },
+            "description": ""
```
Comment (amarzavery) [inline batch/2016-07-01.3.1/swagger/BatchService.json]: empty description not allowed

File: batch/2016-07-01.3.1/swagger/BatchService.json [hunk 2]
```diff
@@ -8253,6 +8253,156 @@
...
+            "description": "The maximum time that the server can spend processing the request, in seconds. The default is 30 seconds.",
+            "x-ms-parameter-grouping": {
+              "postfix": "Options"
+            }
+          },
+          {
+            "name": "client-request-id",
+            "in": "header",
+            "required": false,
+            "type": "string",
```
Comment (amarzavery) [inline batch/2016-07-01.3.1/swagger/BatchService.json]: should we specify the `"format": "uuid"` over here?

**Comment by amarzavery** [comment]
overall LGTM

**Comment by matthchr** [comment]
@amarzavery I will run the linter and fix all of the issues. I'd like to do it in a separate PR though to keep this one clean.  Is that okay?

**Comment by amarzavery** [comment]
sure sounds good. Just resolve the comments made in this PR. Other linter errors and warnings can be fixed in a separate PR.

File: batch/2016-07-01.3.1/swagger/BatchService.json (hunk 2 above)
Comment (matthchr) [inline batch/2016-07-01.3.1/swagger/BatchService.json]: No, our request ID I think is not actually required to be a `uuid`  I will doublecheck with our server guys but last time this came up I think they said we should leave it as string.

**Pull request merged**